sympy/core/mul.py

```diff
@@ -1332,11 +1328,21 @@ def _eval_is_prime(self):
                 r *= arg
```

Comment:
I believe it is more efficient to do `r = Mul(*self.args)` or simply, return `Mul(*self.args).is_prime` -- the same below in `_eval_is_composite`.